src/OrchardCore/Orchard.DisplayManagement/Fluid/FluidViewTemplate.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Fluid;
+using Fluid.Values;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.FileProviders;
+using Newtonsoft.Json.Linq;
+using Orchard.DisplayManagement.Fluid.Filters;
+using Orchard.DisplayManagement.Fluid.Internal;
+using Orchard.DisplayManagement.Fluid.ModelBinding;
+using Orchard.DisplayManagement.Shapes;
+using Orchard.Settings;
+
+namespace Orchard.DisplayManagement.Fluid
+{
+    public class FluidViewTemplate : FluidTemplate<ActivatorFluidParserFactory<FluidViewParser>>
+    {
+        public static readonly string ViewsFolder = "Views";
+        public static readonly string ViewExtension = ".liquid";
+        private static readonly MemoryCache Cache = new MemoryCache(new MemoryCacheOptions());
+
+        static FluidViewTemplate()
+        {
+            TemplateContext.GlobalMemberAccessStrategy.Register<JObject>((obj, name) => obj[name]);
+
+            if (!FluidValue.TypeMappings.TryGetValue(typeof(JObject), out var value))
+            {
+                FluidValue.TypeMappings.Add(typeof(JObject), o => new ObjectValue(o));
+                FluidValue.TypeMappings.Add(typeof(JValue), o => FluidValue.Create(((JValue)o).Value));
+            }
+
+            if (!FluidValue.TypeMappings.TryGetValue(typeof(DateTime?), out value))
```

Comment:
Why is this necessary? Because if the nullable? 
Looks like an issue in Fluid then. I know I need to add support for DateTime as a FluidValue either way.